docs/developer-doc-index.md

```diff
@@ -0,0 +1,4 @@
+# Developer Documentation index
+
+- [Broker Highlevel Architecture](architecture.md)
+- [Adding a new module to repo](adding-a-new-module.md)
```

Comment:
Shall we say "Adding a new module to **the** repo"? Or just "Adding a new module"?

Also, would it sound better as "High-level Architecture of the Broker"?

Reply:
:+1: Yea. It sounds better when put it like that.